hypergan/discriminators/pyramid_discriminator.py

```diff
@@ -108,30 +109,24 @@ def should_ignore_layer(layer, resize):
             xg = tf.concat(axis=0, values=[x_filter_i, g_filter_i])
         else:
             if(config['progressive_enhancement']):
-                xg = tf.concat(axis=0, values=[xs[index], gs[index]])
+                xg = tf.concat(axis=0, values=[xs[i], gs[i]])
 
         if(config['noise'] and xg is not None):
             xg += tf.random_normal(xg.get_shape(), mean=0, stddev=config['noise'], dtype=gan.config.dtype)
   
         if config['progressive_enhancement']:
             net = tf.concat(axis=3, values=[net, xg])
     
-      filter_size_w = 2
-      filter_size_h = 2
-      filter = [1,filter_size_w,filter_size_h,1]
-      stride = [1,filter_size_w,filter_size_h,1]
-      depth = int(int(net.get_shape()[3])*depth_increase)
-      if i ==0:
-          depth = config.first_conv_size
-      if strided:
-          net = conv2d(net, depth, name=prefix+'_expand_layer'+str(i), k_w=3, k_h=3, d_h=filter_size_h, d_w=filter_size_w, regularizer=None)
+      if config.foundation == 'additive':
```

Comment:
Lets just remove this option and change it to always additive.  It's much cheaper and still gives good results.